src/arcsf/constants.py

```diff
@@ -0,0 +1,21 @@
+from pathlib import Path
+
+PROJECT_DIR = Path(__file__, "..", "..", "..").resolve()
+CONFIG_DIR = Path(PROJECT_DIR, "configs")
+
+
+def _get_project_dir(location):
+    """Helper function for creating project config paths.
+
+    Args:
+        location: Directory inside PROJECT_ROOT/configs/ to create path for
+
+    Returns:
+        String giving path of PROJECT_ROOT/configs/location
+    """
+    return str((CONFIG_DIR / location))
```

Comment:
may not be necessary to convert to a string here (but ok to leave if you prefer)

Reply:
Have removed